spec/requests/grades_request_spec.rb

```diff
@@ -0,0 +1,139 @@
+require 'rails_helper'
+
+RSpec.describe "Admin::Grades", type: :request do
+    let(:grade) { create(:grade) }
+    let(:grades) { create_list(:subject, 3) }
+    let!(:admin) { create(:user, :admin) }
+    let(:attr) { { number: '5' , letter: "D" } }
+    let(:wrong_attr) { { number: '5' , letter: "" } }
+    let(:invalid_id) { '998' }
+
+    describe 'GET #index' do
+      context 'with logged-in admin' do
+        before(:each) do
+          sign_in admin
+          get admin_grades_path
+          it 'index status' do
```

Comment:
did you run tests? this will fail because you can't use `it` inside a `before`

Reply:
fixed it/